site2/website/release-notes.md

```diff
@@ -1,5 +1,134 @@
+## Apache Pulsar Release Notes
+
+### 2.5.0 &mdash; 2019-12-06 <a id="2.5.0"></a>
+
+#### PIPs
+
+- [PIP-41] Introduce Protocol Handler API [#5130](https://github.com/apache/pulsar/pull/5130) [#5743](https://github.com/apache/pulsar/pull/5473) 
+- [PIP-45] Pluggable metadata interface [#5330](https://github.com/apache/pulsar/pull/5330) 
+- [PIP-44] Separate schema compatibility checker for producer and consumer [#5227](https://github.com/apache/pulsar/pull/5227)
+- [PIP-43] Producer send messages with different schema [#5141](https://github.com/apache/pulsar/issues/5141) [#5517](https://github.com/apache/pulsar/pull/5517) 
+- [PIP-51] Introduce sticky consumer [#5388](https://github.com/apache/pulsar/pull/5388)
+- [PIP-38] Support batch receive in java client. [#4621](https://github.com/apache/pulsar/pull/4621)
+- [PIP-52] PIP-52: [pulsar-sever] Add support of dispatch throttling relative to publish-rate [#5797](https://github.com/apache/pulsar/pull/5797)
+
+#### Fixes
+
+- [Broker] Avoid retrying deleting namespace when topic is already deleted/fenced [#4665](https://github.com/apache/pulsar/pull/4665)
+- [pulsar-broker] Fix expiry monitor to continue on non-recoverable error [#4818](https://github.com/apache/pulsar/pull/4818) 
+- [pulsar-broker] fix ns-isolation api to fetch policy for specific broker [#5314](https://github.com/apache/pulsar/pull/5314)
+- [broker] external protocols not set to local broker data [#5749](https://github.com/apache/pulsar/pull/5479)
+- [broker]Add handle exception KeeperException.BadVersionException  [#5563](https://github.com/apache/pulsar/pull/5563)
+- Fix message deduplicate issue while using external sequence id with batch produce [#5491](https://github.com/apache/pulsar/pull/5491)
+- Remove cursor while remove non-durable subscription [#5719](https://github.com/apache/pulsar/pull/5719) 
+
+#### Enhancements
+
+- Process requests asynchronously on some REST APIs [4765](https://github.com/apache/pulsar/pull/4765) [4778](https://github.com/apache/pulsar/pull/4778) [4795](https://github.com/apache/pulsar/pull/4795) 
+- Reduce char[] creation on jvm heap [#5055](https://github.com/apache/pulsar/pull/5055)
+- Fixes not owned bundles got selected when broker overloading [#5002](https://github.com/apache/pulsar/pull/5002)
+- [pulsar-broker] support update partition for global topic  [#5306](https://github.com/apache/pulsar/pull/5306)
+- Ensure the handling of PartitionMetadataRequest is async end-to-end [#5307](https://github.com/apache/pulsar/pull/5307)
+- [pulsar-broker] Introduce publish rate-limiting on topic [#3986](https://github.com/apache/pulsar/pull/3986)
+- Allow to automatically assign TCP ports when starting a broker  [#3555](https://github.com/apache/pulsar/pull/3555) 
+- [zookeeper] bump zookeeper to version 3.5.6 [#5043](https://github.com/apache/pulsar/pull/5043)
+- Validate topic name before creating partition/non partition topic via admin cli. [#5148](https://github.com/apache/pulsar/pull/5148)
```

Comment:
You use periods for some sentences while not for others, please check all and keep consistent.

Reply:
sure thanks